index.md

```diff
@@ -71,7 +71,7 @@ hacking/target-cortex-a
 ![](_assets/bmp_v2_3b_iso.jpg)
 
 ## Black Magic Debug Project Goals
-The goal of this project is the creation of an easy to use. Mostly plug and play, JTAG/SWD debugger for embedded microcontrollers. Focusing on professional embedded software developers that prefer retaining control over their build systems and testing environments instead of relying on highly abstracted vendor tools that give the impression of simplicity at the cost of transparency.
+The goal of this project is the creation of an easy to use, mostly plug-and-play, JTAG/SWD debugger for embedded microcontrollers. Focusing on professional embedded software developers that prefer retaining control over their build systems and testing environments instead of relying on highly abstracted vendor tools that give the impression of simplicity at the cost of transparency.
```

Comment:
Given the full stop, `s/Focusing/Focused/` as the tense of the follow-up sentence is wrong.

Reply:
This comment is addressed by reformulating the beginning of the sentence that started with `Focusing`.